### ISSUE TITLE
Make Evil Daisy AI script

### ISSUE THREAD

**Issue by tyomklz** [tyomklz/Kjogi#5]
Ideas? How different it could be?

**Comment by tigrelily** [comment]
For this one I'm thinking of having it be a long range attacker... so one collider for when it "looks" at Kjogi and starts walking towards him and then when he gets in a certain range (another collider?) it can start shooting like fireballs or something from its crazy mouth :p 

Not sure if that's the way you code it but that's how I would ;)

**Comment by tigrelily** [comment]
Also for this one... maybe if can't die by being jumped on my Kjogi. Are we able to throw rocks at all? If not it's fine and we can do the standard death :)

**Comment by tyomklz** [comment]
Yeah, okay, sounds good, we'll need some distant attack enemy type anyway. Yeah, I started writing the 'pick up' script, I'll continue and make throw action as well :)

**Comment by tigrelily** [comment]
Cool... I can draw a two frame fireball attack or something.. it has a fire on it's butt when it attacks anyways :p
